subcommand/server-acl-init/command.go

```diff
@@ -401,51 +348,47 @@ service_prefix "" {
 		}
 	}
 
-	// Create enterprise license token if necessary
-	if c.flagCreateEntLicenseToken {
-		// Create enterprise license policy
-		entLicenseRules := `operator = "write"`
+	return 0
+}
 
-		entLicensePolicy := api.ACLPolicy{
-			Name:        "enterprise-license-token",
-			Description: "Enterprise License Token Policy",
-			Rules:       entLicenseRules,
-		}
+func (c *Command) createACL(name, rules string, consulClient *api.Client) error {
+	// Create policy
+	policy := api.ACLPolicy{
+		Name:        fmt.Sprintf("%s-token", name),
+		Description: fmt.Sprintf("%s Token Policy", name),
+		Rules:       rules,
+	}
 
-		aclEntLicensePolicy, _, err := consulClient.ACL().PolicyCreate(&entLicensePolicy, &api.WriteOptions{})
-		if err != nil {
-			c.UI.Error(fmt.Sprintf("Error creating enterprise license policy: %s", err))
-			return 1
-		}
+	createdPolicy, _, err := consulClient.ACL().PolicyCreate(&policy, &api.WriteOptions{})
+	if err != nil {
+		return errors.New(fmt.Sprintf("Error creating %s policy: %s", name, err))
```

Comment:
```suggestion
                return fmt.Errorf("Error creating %s policy: %s", name, err)
```

Reply:
If the policy is already created, does this fail? Just thinking if one of these steps fail and the job is re-run, if we'll never get past this step.

Reply:
Probably. But we currently only create this on server creation, so if a policy already exists with that name, someone manually changed things as everything was getting created which has larger security implications and should be investigated.